library/lua/gui/widgets/text_area/text_area_content.lua

```diff
@@ -0,0 +1,677 @@
+local gui = require('gui')
+local common = require('gui.widgets.common')
+local Widget = require('gui.widgets.widget')
+local WrappedText = require('gui.widgets.text_area.wrapped_text')
+local HistoryStore = require('gui.widgets.text_area.history_store')
+
+local CLIPBOARD_MODE = {LOCAL = 1, LINE = 2}
+local HISTORY_ENTRY = HistoryStore.HISTORY_ENTRY
+
+TextAreaContent = defclass(TextAreaContent, Widget)
+
+TextAreaContent.ATTRS{
+    text = '',
+    text_pen = COLOR_LIGHTCYAN,
+    ignore_keys = {'STRING_A096'},
```

Comment:
same comment as above -- I think we can change the default to an empty list due to `INTERCEPT_HANDLED_HOTKEYS`